backend/migrations/versions/442dd8f223a9__alembic_sync.py

```diff
@@ -0,0 +1,181 @@
+"""_describe_changes_shortly
+
+Revision ID: 442dd8f223a9
+Revises: f6cd4ba7dd8d
+Create Date: 2024-02-08 13:19:22.898697
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = '442dd8f223a9'
+down_revision = 'f6cd4ba7dd8d'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_table('dataset_table_profiling_job')
+    op.drop_table('group_member')
+    op.drop_table('redshiftcluster')
+    op.drop_table('dataset_quality_rule')
+    op.drop_table('tag')
+    op.drop_table('redshiftcluster_datasettable')
+    op.drop_table('item_tags')
+    op.drop_table('redshiftcluster_dataset')
+    op.alter_column('datapipelineenvironments', 'region',
+                    existing_type=sa.VARCHAR(),
+                    nullable=True)
+    op.drop_constraint('dataset_bucket_datasetUri_fkey', 'dataset_bucket', type_='foreignkey')
+    op.create_foreign_key(None, 'dataset_bucket', 'dataset', ['datasetUri'], ['datasetUri'], ondelete='CASCADE')
+    op.alter_column('environment_parameters', 'paramValue',
+                    existing_type=sa.VARCHAR(),
+                    nullable=True)
+    op.create_foreign_key(None, 'environment_parameters', 'environment', ['environmentUri'], ['environmentUri'])
+    op.alter_column('sagemaker_studio_domain', 'environmentUri',
```

Comment:
Think we can change our model to make environmentUri not nullable for `sagemaker_studio_domain`